.github/workflows/build.yml

```diff
@@ -78,93 +110,193 @@ jobs:
+              repo: context.repo.repo,
+              ref: 'refs/tags/v${{ steps.version.outputs.version }}',
+              sha: context.sha
+            })
 
-      - name: Send message to skype
-        uses: Eloco/docker-action-send-skype@v2
-        if: always()
+      - name: Tag to trigger PR merge on increment / release branch
+        if: |-
+          ${{ always() && 
+          (contains(github.head_ref, 'increment') || contains(github.head_ref, 'release')) && 
+          job.status == 'success' }}
+        uses: actions/github-script@v5
         with:
-          skype_username: ${{ secrets.SKYPE_USERNAME }}
-          skype_password: ${{ secrets.SKYPE_PASSWORD }}
-          skype_ids: 19:0503749da2e44ed9910046678012bdf7@thread.skype
-          send_msg_path: build.msg
+          github-token: ${{ secrets.OSS_PAT }}
+          script: |
+            github.rest.git.createRef({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              ref: 'refs/tags/merge-pr/${{ steps.current-pr.outputs.pr }}',
+              sha: context.sha
+            })
+
+      - name: Tag to trigger release on master / develop branch
+        if: |-
+          ${{ always() && 
+          (contains(github.head_ref, 'develop') || contains(github.ref, 'develop') || 
+            contains(github.head_ref, 'master') || contains(github.ref, 'master')) && 
+          job.status == 'success' }}
+        uses: actions/github-script@v5
+        with:
+          github-token: ${{ secrets.OSS_PAT }}
+          script: |
+            github.rest.git.createRef({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              ref: 'refs/tags/create-release/${{ steps.version.outputs.version }}',
+              sha: context.sha
+            })
+
+      - name: Create message
+        id: message
+        if: ${{ always() }}
+        run: |-
+          if [ "${{ job.status }}" == "success" ]; then
+            message=":check_mark_button: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+            message="${message} Version: ${{ steps.version.outputs.version }}"
+          fi
+          if [ "${{ job.status }}" == "failure" ]; then
+            message=":cross_mark: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"    
+          fi
+          if [ "${{ job.status }}" == "canceled" ]; then
+            message=":cross_mark: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"    
+          fi
+          message="${message} Commit: ${{ steps.context.outputs.commitMsg }}"
+          message="${message} User: ${{ github.actor }}"
+          echo "::set-output name=message::${message}"
+
+      - name: Write message to file
+        run: |-
+          echo "${{ steps.message.outputs.message }}"
+          echo "${{ steps.message.outputs.message }}" > build.msg
+
+#      - name: Create success message
```

Comment:
Success / Failure message in test-workflow is commented out just to do not make unneccessary Noise in skype :) In this project it can be switched on.